### DISCUSSION TITLE
虚拟滚动树例子中，treeRef.value为null

### DISCUSSION THREAD

**Discussion by sasadaisy** [DevCloudFE/vue-devui#1463] [Q&A] [answered]
虚拟滚动树例子中，treeRef.value为null

**Comment by kagol** [comment]
@sasadaisy 是这个例子吗？
[https://vue-devui.github.io/components/tree/#虚拟滚动](https://vue-devui.github.io/components/tree/#%E8%99%9A%E6%8B%9F%E6%BB%9A%E5%8A%A8)

**Comment by kagol** [comment, marked as answer]
@sasadaisy 我尝试在 Playground 中打印了 treeRef，确实有有值的哦，你可以尝试在 nextTick 或者 onMounted 事件中打印。

```
  nextTick(() => {
    console.log('treeRef:', treeRef);
  });
```


[Tree 虚拟滚动例子，测试 treeRef 的值](https://devcloudfe.github.io/devui-playground/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8aDE+XG4gICAgSGVsbG8sIERldlVJIVxuICA8L2gxPlxuICA8ZC1idXR0b24gdmFyaWFudD1cInNvbGlkXCIgc3R5bGU9XCJtYXJnaW4tcmlnaHQ6IDhweFwiPlByaW1hcnk8L2QtYnV0dG9uPlxuICA8ZC1idXR0b24gOmRpc2FibGVkPVwidHJ1ZVwiPkRpc2FibGVkPC9kLWJ1dHRvbj5cbiAgPGQtdHJlZSA6ZGF0YT1cImRhdGFcIiA6aGVpZ2h0PVwiMzAwXCIgcmVmPVwidHJlZVJlZlwiPjwvZC10cmVlPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbiAgaW1wb3J0IHsgZGVmaW5lQ29tcG9uZW50LCByZWYsIG9uTW91bnRlZCwgbmV4dFRpY2sgfSBmcm9tICd2dWUnO1xuaW1wb3J0IHsgc2V0dXBEZXZ1aSB9IGZyb20gJy4vZGV2dWkuanMnXG5cbi8vIGRvbid0IHJlbW92ZVxuc2V0dXBEZXZ1aSgpXG4gIFxuIFx0Y29uc3QgdHJlZVJlZiA9IHJlZihudWxsKTtcbiAgY29uc3QgZGF0YSA9IHJlZihbXG4gICAgLi4uQXJyYXkuZnJvbSh7IGxlbmd0aDogMTAwIH0pLm1hcCgoXywgaW5kZXgpID0+ICh7XG4gICAgICBsYWJlbDogJ1BhcmVudCBub2RlICcgKyBpbmRleCxcbiAgICAgIGNoaWxkcmVuOlxuICAgICAgaW5kZXggJSAyID09PSAwXG4gICAgICA/IEFycmF5LmZyb20oeyBsZW5ndGg6IDEwIH0pLm1hcCgoXywgaW5kZXgyKSA9PiAoe1xuICAgICAgICBsYWJlbDogJ0xlYWYgbm9kZSAnICsgaW5kZXggKyAnLScgKyBpbmRleDIsXG4gICAgICB9KSlcbiAgICAgIDogdW5kZWZpbmVkLFxuICAgIH0pKSxcbiAgXSk7XG4gIFxuICBuZXh0VGljaygoKSA9PiB7XG4gICAgY29uc29sZS5sb2coJ25leHRUaWNrIHRyZWVSZWY6JywgdHJlZVJlZik7XG4gIH0pO1xuXG4gIG9uTW91bnRlZCgoKSA9PiB7XG4gICAgY29uc29sZS5sb2coJ3RyZWVSZWYnLCB0cmVlUmVmKTtcbiAgICBcbiAgICB0cmVlUmVmLnZhbHVlLnRyZWVGYWN0b3J5LmV4cGFuZEFsbE5vZGVzKCk7XG4gIH0pO1xuPC9zY3JpcHQ+IiwiaW1wb3J0LW1hcC5qc29uIjoie1wiaW1wb3J0c1wiOntcImRldnVpLXRoZW1lXCI6XCJodHRwczovL3VucGtnLmNvbS9kZXZ1aS10aGVtZUBsYXRlc3QvaW5kZXguZXMuanNcIixcInZ1ZS1kZXZ1aVwiOlwiaHR0cHM6Ly91bnBrZy5jb20vdnVlLWRldnVpQGxhdGVzdC92dWUtZGV2dWkuZXMuanNcIixcInZ1ZVwiOlwiaHR0cHM6Ly91bnBrZy5jb20vdnVlQGxhdGVzdC9kaXN0L3Z1ZS5lc20tYnJvd3Nlci5qc1wiLFwidnVlLXJvdXRlclwiOlwiaHR0cHM6Ly91bnBrZy5jb20vdnVlLXJvdXRlckBsYXRlc3QvZGlzdC92dWUtcm91dGVyLmVzbS1icm93c2VyLmpzXCIsXCJAZmxvYXRpbmctdWkvZG9tXCI6XCJodHRwczovL3VucGtnLmNvbS9AZmxvYXRpbmctdWkvZG9tQGxhdGVzdC9kaXN0L2Zsb2F0aW5nLXVpLmRvbS5lc20uanNcIixcIkBmbG9hdGluZy11aS9jb3JlXCI6XCJodHRwczovL3VucGtnLmNvbS9AZmxvYXRpbmctdWkvY29yZUBsYXRlc3QvZGlzdC9mbG9hdGluZy11aS5jb3JlLmVzbS5qc1wiLFwiQHZ1ZXVzZS9jb3JlXCI6XCJodHRwczovL3VucGtnLmNvbS9AdnVldXNlL2NvcmVAbGF0ZXN0L2luZGV4Lm1qc1wiLFwiQHZ1ZS9kZXZ0b29scy1hcGlcIjpcImh0dHBzOi8vdW5wa2cuY29tL0B2dWUvZGV2dG9vbHMtYXBpQGxhdGVzdC9saWIvZXNtL2luZGV4LmpzXCIsXCJAdnVldXNlL3NoYXJlZFwiOlwiaHR0cHM6Ly91bnBrZy5jb20vQHZ1ZXVzZS9zaGFyZWRAbGF0ZXN0L2luZGV4Lm1qc1wiLFwidnVlLWRlbWlcIjpcImh0dHBzOi8vdW5wa2cuY29tL3Z1ZS1kZW1pQGxhdGVzdC9saWIvaW5kZXgubWpzXCJ9fSJ9)

![image](https://user-images.githubusercontent.com/9566362/208210415-7e42bf56-a004-44f6-9872-922aa5f928ac.png)
